playbook/app/pb_kits/playbook/pb_map/docs/_map_default.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useRef, useEffect } from 'react'
+import { Map } from '../../'
+
+import maplibregl from 'maplibre-gl'
+
+const MapDefault = () => {
+
+  const mapContainerRef = useRef(null)
+
+    useEffect(() => {
+      if (!maplibregl.supported()) {
+        alert('Your browser does not support MapLibre GL');
+        } else {
+         const map = new maplibregl.Map({
+            container: mapContainerRef.current,
+            style: 'https://basemaps.cartocdn.com/gl/positron-gl-style/style.json',
+            center: [-75.379143, 39.831200],
+            zoom: 13,
+        })
+        //set marker/pin
+        /* eslint-disable-next-line */
+        const marker = new maplibregl.Marker({
```

Comment:
You might even want to separate this out a little into it's own object. That will make this useEffect a little more readable.